haskell-customize.el

```diff
@@ -115,19 +115,28 @@ when showing type information about symbols."
 
 (defcustom haskell-process-path-ghci
   "ghci"
-  "The path for starting ghci."
+  "The path for starting ghci.
+This can either be a single string or a list of strings, where the
+first elements is a string and the remaining elements are arguments,
+which will be prepended to `haskell-process-args-gci`."
```

Comment:
Gracjan Polak <notifications@github.com> writes:

>> @@ -115,19 +115,28 @@ when showing type information about symbols."
>>  
>>  (defcustom haskell-process-path-ghci
>>    "ghci"
>> -  "The path for starting ghci."
>> +  "The path for starting ghci.
>> +This can either be a single string or a list of strings, where the
>> +first elements is a string and the remaining elements are arguments,
>> +which will be prepended to `haskell-process-args-gci`."
>
> Typo: s/gci/ghci/g. Same with other docstrings. Thanks.

Done.

-- 
Regards,
Mike